openapi/SwarmDebug.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 2.1.0
+  version: 2.1.1
```

Comment:
There has been no change in the debug API, so there is no need to change the version.